spec/requests/posts_spec.rb

```diff
@@ -2,16 +2,12 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
```

Comment:
- [ ] https://github.com/DevKaranJ/Blog/blob/1b443ff641535651becf7513e2f1c578ee7e5741/spec/requests/users_spec.rb#L3-L16
 
  - Please make sure to add other missing tests for the user index and user show actions as requested by me previously and all tests should pass without any errors.
 
![image](https://github.com/DevKaranJ/Blog/assets/83803414/b01739d6-7e6d-45f6-944f-6d57cadd8e56)